sycl/include/CL/sycl/builtins_esimd.hpp

```diff
@@ -0,0 +1,95 @@
+//==----------- builtins_esimd.hpp - SYCL ESIMD built-in functions ---------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl/detail/boolean.hpp>
+#include <CL/sycl/detail/builtins.hpp>
+#include <CL/sycl/detail/common.hpp>
+#include <CL/sycl/detail/generic_type_traits.hpp>
+#include <CL/sycl/types.hpp>
+#include <sycl/ext/intel/experimental/esimd/detail/math_intrin.hpp>
+
+// TODO Decide whether to mark functions with this attribute.
+#define __NOEXC /*noexcept*/
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+
+#define __ESIMD_NS ext::intel::experimental::esimd
```

Comment:
this one is already defined:
`sycl/include/sycl/ext/intel/experimental/esimd/common.hpp:#define __ESIMD_NS sycl::ext::intel::experimental::esimd`

Reply:
Done.